movement/plots/occupancy.py

```diff
@@ -0,0 +1,185 @@
+
+    >>> plot_occupancy(
+    ...     positions,
+    ...     # use only one keypoint in computation of centroid
+    ...     keypoints="centroid",
+    ...     # only aggregate for two individuals
+    ...     individuals=[
+    ...         "AEON3B_TP1",
+    ...         "AEON3B_TP2",
+    ...     ],
+    ...     # fix the number of bins to use
+    ...     bins=[30, 20],
+    ...     # set the minimum count (bins with a lower count show as 0 count).
+    ...     # This effectively only displayed bins that were visited at least
+    ...     # twice.
+    ...     cmin=1,
+    ...     # Normalise the plot, scaling the counts to [0, 1]
+    ...     norm=True,
+    ... )
+
+    See Also
+    --------
+    matplotlib.pyplot.Axes.hist2d : The underlying plotting function.
+
+    """
+    # Collapse dimensions if necessary
+    data = da.copy(deep=True)
+    if "keypoints" in da.dims:
+        if keypoints is not None:
+            data = data.sel(keypoints=keypoints)
+        # A selection of just one keypoint automatically drops the keypoints
+        # dimension, hence the need to re-check this here
+        if "keypoints" in data.dims:
+            data = data.mean(dim="keypoints", skipna=True)
+    if "individuals" in da.dims and individuals is not None:
+        data = data.sel(individuals=individuals)
+
+    # We need to remove NaN values from each individual, but we can't do this
+    # right now because we still potentially have a (time, space, individuals)
+    # array and so dropping NaNs along any axis may remove valid points for
+    # other times / individuals.
+    # Since we only care about a count, we can just unravel the individuals
+    # dimension and create a "long" array of points. For example, a (10, 2, 5)
+    # time-space-individuals DataArray becomes (50, 2).
+    if "individuals" in data.dims:
+        data = data.stack(
+            {"new": ("time", "individuals")}, create_index=False
+        ).rename({"new": "time"})
+    # We should now have just the relevant time-space data,
+    # so we can remove time-points with NaN values.
+    data = data.dropna(dim="time", how="any")
+
+    # This makes us agnostic to the planar coordinate system.
+    x_coord = data["space"].values[0]
+    y_coord = data["space"].values[1]
+
+    # Inherit our defaults if not otherwise provided
+    for key, value in DEFAULT_HIST_ARGS.items():
+        if key not in kwargs:
+            kwargs[key] = value
+    # Now it should just be a case of creating the histogram
+    if ax is not None:
+        fig = ax.get_figure()
+    else:
+        fig, ax = plt.subplots()
+    counts, xedges, yedges, hist_image = ax.hist2d(
+        data.sel(space=x_coord), data.sel(space=y_coord), **kwargs
+    )
+    colourbar = fig.colorbar(hist_image, ax=ax)
+    colourbar.solids.set(alpha=1.0)
+
+    space_unit = data.attrs.get("space_unit", "pixels")
+    ax.set_xlabel(f"{x_coord} ({space_unit})")
+    ax.set_ylabel(f"{y_coord} ({space_unit})")
```

Comment:
For reasons mentioned in issue #417, I suggest ditching the space unit and just labelling the axes as "x" and "y".